tests/dm_control/test_dm_control.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from dm_control import suite
+
+from garage.envs import normalize
+from garage.envs.dm_control import DmControlEnv
+
+
+def run_task(domain_name, task_name):
+    print("run: domain %s task %s" % (domain_name, task_name))
+    dm_control_env = normalize(
+        DmControlEnv(
+            domain_name=domain_name,
+            task_name=task_name,
+            plot=True,
+            width=600,
+            height=400),
+        normalize_obs=False,
+        normalize_reward=False)
+
+    time_step = dm_control_env.reset()
+    action_spec = dm_control_env.action_space
+    for _ in range(5):
```

Comment:
Isn't there a helper which does exactly this stepping test in Angel's PR?

Reply:
Yes, I will use his implementation or remove this file altogether.